mobilitypath_publisher/include/mobilitypath_publisher.h

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
+/*
+ * Copyright (C) 2019-2020 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+#include <string>
+#include <vector>
+#include <boost/uuid/uuid_generators.hpp>
+#include <boost/uuid/uuid_io.hpp>
+#include <tf2_ros/transform_listener.h>
+#include <boost/shared_ptr.hpp>
+#include <carma_utils/CARMAUtils.h>
+#include <cav_msgs/TrajectoryPlan.h>
+#include <cav_msgs/MobilityPath.h>
+#include <geometry_msgs/PoseStamped.h>
+
+namespace mobilitypath_publisher
+{
+
+    class MobilityPathPublication
+    {
+
+    public:
+
+        MobilityPathPublication();
+
+        // general starting point of this node
+        void run();
+
+        // local copy of pose
+        boost::shared_ptr<geometry_msgs::PoseStamped const> current_pose_;
+
+        cav_msgs::MobilityPath mobilityPathMessageGenerator(cav_msgs::TrajectoryPlan trajectory_plan);
+        
+
+    private:
+
+        // node handles
+        std::shared_ptr<ros::CARMANodeHandle> nh_, pnh_;
+
+
+        // ROS publisher
+        ros::Publisher  path_pub_;
+
+        // ROS subscribers
+        ros::Subscriber traj_sub_;
+        ros::Subscriber pose_sub_;
+        ros::Subscriber accel_sub_;
+
+        // ROS publishers
+        ros::Publisher mob_path_pub_;
+
+        cav_msgs::TrajectoryPlan latest_trajectory;
```

Comment:
Should add an ending "_" to be consistent with other member variables in this file.

Reply:
fixed